modules/sugar/src/main/ts/ephox/sugar/api/search/TransformFind.ts

```diff
@@ -20,11 +22,12 @@ const ancestor = function <A> (scope: Element<DomNode>, transform: (e: Element<D
   return Option.none<A>();
 };
 
-const closest = function <A> (scope: Element<DomNode>, transform: (e: Element<DomNode>) => Option<A>, isRoot?: (e: Element<DomNode>) => boolean): Option<A> {
+const closest = <A> (scope: Element<DomNode>, transform: (e: Element<DomNode>) => Option<A>, isRoot?: (e: Element<DomNode>) => boolean): Option<A> => {
```

Comment:
This almost definitely had a bug, as it had `isRoot` as being optional but then tried to use it directly without checking if it was undefined a couple lines below.

Reply:
Should we log a ticket to go through at some point and make it not optional? Being Sugar it'd be a mess, but it might be a good easy task for someone at some point.

Reply:
Logged as TINY-4834